h2o-py/tests/testdir_algos/xgboost/pyunit_H2OXGBoost_native_XGBoost_mixed_binomial_compare_large_dense_NOPASS_multinode.py

```diff
@@ -56,7 +56,7 @@ def comparison_test_dense():
 
     # train the native XGBoost
     nativeTrain = pyunit_utils.convertH2OFrameToDMatrix(trainFile, y, enumCols=enumCols)
-    nativeModel = xgb.trainxgb.train(params=nativeParam,
```

Comment:
NOPASS ?

Reply:
All tests are running in single node mode in order to pass.  Hence, they have been running all these time thanks to Michal R.  However, if you do not put NOPASS in the name, it will be called in general running mode and it will fail.